src/litserve/loops.py

```diff
@@ -399,18 +399,23 @@ def inference_worker(
     max_batch_size: int,
     batch_timeout: float,
     stream: bool,
-    workers_setup_status: Dict[str, bool],
+    workers_setup_status: Dict[int, str],
```

Comment:
can we have docs about the meaning or samples?

Reply:
did you mean docstrings?

Reply:
Yes

Reply:
great suggestion! although this is a private function so users won't be using it directly but might be useful for contributors.